packages/checkbox/src/Checkbox.stories.tsx

```diff
@@ -123,3 +128,18 @@ export const Controlled = () => {
 
   return <Checkbox isChecked={value} onChange={handleChange} />
 }
+
+export const CheckboxGroupExample = () => {
+  return (
+    <CheckboxGroup
+      spacing={[2, 4, 6]}
+      direction={["column", "row"]}
+      defaultValue={["one", "two"]}
```

Comment:
Oooohhh, I didn't realize `defaultValue` could be responsive. That's neat.

Reply:
Haha. It's not that it's responsive but it's because it's a CheckboxGroup. The internal state value is an array since multiple checkboxes can be checked at a time 😃

Reply:
@segunadebayo Oh, derp. Yeah, responsive value really wouldn't make sense. It was still early in the morning 😴